lib/mongoose_push_web/schemas/api_v1_request_push.ex

```diff
@@ -0,0 +1,32 @@
+defmodule MongoosePushWeb.Schemas.APIv1.Request.Push do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "APIv1.Request.Push",
+    description: "Push notification request schema",
+    type: :object,
+    properties: %{
+      service: %Schema{
+        type: :string,
+        description: "Push notification service",
+        format: :string,
+        enum: ["fcm", "apns"]
+      },
+      body: %Schema{type: :string, description: "Body of the notification", format: :string},
+      title: %Schema{type: :string, description: "Title of the notification", format: :string},
+      badge: %Schema{type: :integer, format: :int32},
+      click_action: %Schema{type: :string},
+      tag: %Schema{type: :string},
+      topic: %Schema{type: :string},
+      data: %Schema{type: :string},
+      mode: %Schema{type: :object, enum: [:prod, :dev]}
```

Comment:
In Maru it was `Atom`, what is not available here, so I tried using `:object`.

Reply:
I corrected the type to `string` to comply with JSON input standards, but still would like somebody to glance at it and check if that's correct.